### ISSUE TITLE
Electron 3.0.x arm tray icon bugs

### ISSUE THREAD

**Issue by lauorozco13** [electron/electron#13980]
* Electron Version: 3.0.0
* Operating System (Platform and Version): Ubuntu Mate Armv7l 
* Last known working Electron version: 2.6.0

**Expected Behavior**
Mouse click on tray icon - activates window.

**Actual behavior**
Shows tray icon popup menu.

**Screenshots**
![captura de pantalla 2018-08-08 10-58-07](https://user-images.githubusercontent.com/26728517/43849233-8155f3c2-9b24-11e8-835d-3e17ca071de1.png)

**Comment by sofianguy** [comment]
Can you provide us with steps to reproduce this issue?

**Comment by ckerr** [comment]
I agree with @sofianguy -- would it be possible for you to fork [electron-quick-start](https://github.com/electron/electron-quick-start) for a small app that reproduces the issue by itself?

Standalone test cases make fixing issues go more smoothly: it ensure everyone's looking at the same issue, it removes all unneccessary variables from the equation, and it can also provide the basis for automated regression tests.

Thanks in advance!

**Comment by ckerr** [comment]
(also, did you mean to type `2.0.6` instead of `2.6.0`?)
